### ISSUE TITLE
Usability Issues. Large font size settings. Some texts are cropped others break the layout

### ISSUE THREAD

**Issue by helloantoine** [covid-alert-ny/covid-green-app#226]
Issue in the visual. Solutions in the list below:
- [ ] Onboarding. How it works. Cap the title as well as the Previous/Next text sizes 
- [x] Cap line height in graphs of the Data tab
- [x] Selectors are close to unusable with very large fonts and keyboard open. Ca the text size.
- [x] My health log. Symptoms list breaks the card. Wrap the text (and cap the text size if needed)

@ahurworth 

![Image from iOS (2)](https://user-images.githubusercontent.com/16833643/91996759-454bd700-ed31-11ea-98e4-f178abce2a64.png)
![Image from iOS (3)](https://user-images.githubusercontent.com/16833643/91996500-f2721f80-ed30-11ea-810f-1772a02a35bd.png)
![Image from iOS (1)](https://user-images.githubusercontent.com/16833643/91996878-6280a580-ed31-11ea-915d-e1011e963beb.png)
![Screenshot 2020-09-02 at 15 27 50](https://user-images.githubusercontent.com/16833643/91996428-dff7e600-ed30-11ea-9a8e-eacd5cd5f822.png)

**Comment by JohnAGill** [comment]
This will be in the next build

**Comment by helloantoine** [comment]
I can't see this in Build 25.

**Comment by floridemai** [comment]
@JohnAGill Can you check this again please?

**Comment by helloantoine** [comment]
Tested in Build 40. Most of it fixed. However:
How it works title is not fixed. See screen.
County, age, ... selectors. Some of them have text on the close button. See screen.

![Image from iOS (2)](https://user-images.githubusercontent.com/16833643/93129942-b4c6ac80-f6c9-11ea-80c2-5969cae346c7.png)
![Image from iOS (1)](https://user-images.githubusercontent.com/16833643/93129936-b2fce900-f6c9-11ea-996c-9d463c246f69.png)

**Comment by JohnAGill** [comment]
Made these last few fixes should be in the next build

**Comment by helloantoine** [comment]
@JohnAGill Everything good EXCEPT the how it works title. Can you double check please?
![Image from iOS](https://user-images.githubusercontent.com/16833643/93451817-8fa38b00-f8cf-11ea-901b-4ed5d0df1707.png)

**Comment by helloantoine** [comment]
Issue still persistent on Build 43

**Comment by JohnAGill** [comment]
This should be fix and in the next build

**Comment by denisedresbach** [comment]
Testing this in 1.1.0.52 
iOS - Changed to largest display size and can see How it works is no longer cut off PASS
Android - Also looks good at largest font and largest display PASS
Took a look at the other items in the list and they look good still.
